app/views/admin/ofsted/pending.html.erb

```diff
@@ -9,7 +9,7 @@
     <% @requests.each do |s| %>
       <li class="pending-panel">
         <div>
-          <h2><%= link_to s.setting_name.truncate(25), admin_ofsted_path(s) %></h2>
+          <h2><%= link_to s.setting_name&.truncate(25) || 'Deleted', admin_ofsted_path(s) %></h2>
```

Comment:
Could you explain this please? Is it saying that if no name is present then display 'Deleted'? Is it possible that an ofsted item might jus not have a setting name?

Reply:
Good spot :+1: this should have been removed, have fixed now